### DISCUSSION TITLE
Unable to update nuclei templates via CLI

### DISCUSSION THREAD

**Discussion by aravindb26** [projectdiscovery/nuclei#4764] [General]
### Nuclei version:
**Nuclei Engine Version: v3.1.10**

### Current Behavior:
**Unable to update nuclei templates via CLI**
**command used: `nuclei --update-templates --silent`**

### Expected Behavior:
**It should update all new templates added in the repo as soon as we use `nuclei --update-templates --silent` command**

### Steps To Reproduce:

1. Use `nuclei --update-templates --silent` to update for any new templates added

2. But still we can't find newly updated templates in the repo

### Anything else:
**<img width="683" alt="nuclei_up" src="https://github.com/projectdiscovery/nuclei/assets/89305482/83e66179-8f28-4682-8642-47b618cda42b">**

**Comment by olearycrew** [comment]
@aravindb26 that template in particular was merged only a few hours ago, and thus isn't included yet in the default nuclei templates but will be in the next release of nuclei-templates.

https://github.com/projectdiscovery/nuclei-templates/commits/main/http/exposed-panels/proofpoint-protection-server-panel.yaml

**Reply by aravindb26** [reply]
So it will be updated only if you release it officially?

**Reply by olearycrew** [reply]
Yes - when an official release of templates is cut that is when nuclei automatically updates with them. See "Templates exist on GitHub but are not running with Nuclei?" for more info: https://docs.projectdiscovery.io/templates/faq

**Comment by dogancanbakir** [comment]
This discussion closed automatically due to inactivity. Feel free to reopen or start new if still relevant.
